libV2/schemaUtils.js

```diff
@@ -736,6 +736,71 @@ let QUERYPARAM = 'query',
     return schema;
   },
 
+  /** process and resolves types from nested schema structure **/
```

Comment:
Can you write a proper JSDOC for this ? And also give a better explanation of this function.